lib/app/modules/bulk_data/bulk_data_capability_statement_sequence.rb

```diff
@@ -65,32 +65,29 @@ def assert_operation(op_name); end
           link 'http://hl7.org/fhir/us/core/2019Jan/CapabilityStatement-us-core-server.html'
           desc %(
 
-            FHIR provides multiple [representation formats](https://www.hl7.org/fhir/DSTU2/formats.html) for resources, including JSON and XML.
-            Argonaut profiles require servers to use the JSON representation:
+            FHIR provides multiple [representation formats](https://www.hl7.org/fhir/formats.html) for resources, including JSON and XML.
+            US Core profiles require servers to use the [JSON representation](https://www.hl7.org/fhir/json.html):
 
-            ```
-            The Argonaut Data Query Server shall support JSON resource format for all Argonaut Data Query interactions.
-            ```
-            [http://hl7.org/fhir/us/core/2019Jan/CapabilityStatement-us-core-server.html](http://hl7.org/fhir/us/core/2019Jan/CapabilityStatement-us-core-server.html)
+            [```The US Core Server **SHALL** Support json source formats for all US Core interactions.```](https://build.fhir.org/ig/HL7/US-Core-R4/CapabilityStatement-us-core-server.html)
 
-            The FHIR capability interaction require servers to describe which formats are available for clients to use.  The server must
-            explicitly state that JSON is supported. This is located in the [format element](https://www.hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.format)
+            The FHIR capability interaction require servers to describe which formats are available for clients to use.
+            The server must explicitly state that JSON is supported.  This is located in the
+            [format element](https://www.hl7.org/fhir/capabilitystatement-definitions.html#CapabilityStatement.format)
             of the Capability Resource.
 
-            This test checks that one of the following values are located in the [format field](https://www.hl7.org/fhir/DSTU2/json.html).
+            This test checks that one of the following values are located in the format field.
 
             * json
             * application/json
-            * application/json+fhir
+            * application/fhir+json
 
-            Note that FHIR changed the FHIR-specific JSON mime type to `application/fhir+json` in later versions of the specification.
+            This test supports the [Bulk Data US Core Data for Interoperability Use Case](https://build.fhir.org/ig/HL7/bulk-data/#us-core-data-for-interoperability)
```

Comment:
Is this requirement actually stated somewhere in that IG that we can link to, or is it being inferred?

Reply:
I didn't really see a hard requirement, just the use case and was attempting to update the bulk data details as best I could.

@yunwwang any thoughts here or should this test be removed

Reply:
Also happy to remove these updates if they need more discussion so that the original typo fix can get in.

Reply:
removed the bulk data IG reference.